src/main.cpp

```diff
@@ -184,7 +184,14 @@ static bool initSettings(SettingsFile *settings, QLockFile **lockFile, QString &
 
     QDir dir(configPath);
     if (!dir.exists() && !dir.mkpath(QStringLiteral("."))) {
-        errorMessage = QStringLiteral("Cannot create directory: %1").arg(dir.path());
+#ifdef Q_OS_MAC
+        QString absolutePath = QFileInfo(QLatin1String(".")).absoluteFilePath();
+        if (absolutePath.startsWith(QLatin1String("/private")) ||
+            absolutePath.startsWith(QLatin1String("/Volumes")))
+            errorMessage = QStringLiteral("Please move Ricochet outside of the DMG before attempting to run it.");
```

Comment:
We could probably nitpick and debate the language for awhile. I don't know if this error means anything more to most people who run into this problem than the current one does. I don't know if it should be translated.

I'm more inclined to not be picky about it, though.